webapp/channels/src/components/login/login.tsx

```diff
@@ -129,22 +131,31 @@ const Login = ({onCustomizeHeader}: LoginProps) => {
     const enableSignInWithEmail = EnableSignInWithEmail === 'true';
     const enableSignInWithUsername = EnableSignInWithUsername === 'true';
     const enableSignUpWithEmail = EnableSignUpWithEmail === 'true';
-    const enableSignUpWithGitLab = EnableSignUpWithGitLab === 'true';
```

Comment:
This logic needs to remain the way it was before, we can not assign this logic to `true` directly. We need to check if that config value is actually enabled.

Reply:
damn, sorry my bad, should've check this before raising the PR